content/en/integrations/guide/aws-integration-manual-setup.md

```diff
@@ -0,0 +1,87 @@
+3. Copy this value to your clipboard or notepad.  
+  **Note: Do not close the integration tile or the Datadog application page**, as this causes the external ID value to reset.
+
+### AWS IAM Policy for Datadog
+Create an IAM policy for the Datadog role in your AWS account with [all permissions][4] to take advantage of every AWS integration offered by Datadog. As other components are added to an integration, these permissions may change. If you would like to use Datadog's [Cloud Security Posture Management product][5], you must also add additional permissions by attaching the [AWS SecurityAudit Policy][6] to the role.
+
+4. Create a new policy in the AWS [IAM Console][7].
+5. Select the `JSON` tab. Paste the [permission policies][4] in the textbox. 
+6. Add the [AWS SecurityAudit Policy][6] permissions if you would like to use Datadog's [Cloud Security Posture Management product][5].
+7. Click `Next: Tags` and `Review policy`.
+8. Name the policy `DatadogAWSIntegrationPolicy` or one of your own choosing, and provide an apt description.
+9. Click `Create policy`.
+
+### AWS IAM Role for Datadog
+Create an IAM role for Datadog to use the permissions defined in the IAM policy.
+
+10. Create a new role in the AWS [IAM Console][8].
+11. Select `Another AWS account` for the Role Type.
+12. For Account ID, enter `464622532012` (Datadog's account ID). This means that you are granting Datadog access to your AWS data.
+13. Select `Require external ID` and enter the one generated in [AWS External ID](#generate-an-external-id). 
+Make sure you leave **Require MFA** disabled. 
+14. Click `Next`.
+15. If you've already created the policy, search for it on this page and select it. Otherwise, click `Create Policy`, which opens in a new window, and follow the instructions from [AWS IAM Policy for Datadog](#aws-iam-policy-for-datadog). 
+16. Click `Next: Tags` and `Next: Review`.
+17. Give the role a name such as `DatadogAWSIntegrationRole`, as well as an apt description. 
+18. Click `Create Role`.
+
+### Complete the setup in Datadog
+
+19. Returning to the [AWS integration tile][2] page in your Datadog account that you had open in another tab, enter your AWS Account ID **without dashes**, for example: `123456789012`. Your Account ID can be found in the ARN of the role created for Datadog.
+20. Enter the name of the created role. **Note:** The role name you enter in the integration tile is case sensitive and must exactly match the role name created on the AWS side.
+21. If there is a [Datadog is not authorized to perform sts:AssumeRole][9] error, make sure your AWS trust policy's `sts:ExternalId:` matches the [`AWS External ID`](#generate-an-external-id) previously created in the integration tile.
+22. Optionally, customize the AWS services to collect metrics from on the left side of the dialog.
```

Comment:
Let's remove all these "Optionally" steps. Re-reading this, I think it is a bit confusing without explaining more about what/where each of these are, so I think we should just remove